### DISCUSSION TITLE
Zoom authentication fails

### DISCUSSION THREAD

**Discussion by regelmol** [HCL-TECH-SOFTWARE/domino-online-meeting-integration#74] [General]
I'm getting this error, when I try to authorize DOMI to Zoom.

"Unable to install this app without the developer's account. Please contact the app developer to install."

I see this has been discussed in April yet and Zoom needs to approve the app. Any timeframe when this will happen?

Thanks,
Oliver

**Comment by spanky762** [comment]
Hi Oliver.  
We are still waiting on this.  Unfortunately at this time I do not know when they will issue the approval.   
-Devin.
